sda-commons-client-jersey-example/src/main/java/org/sdase/commons/client/jersey/JerseyClientExampleApplication.java

```diff
@@ -1,10 +1,12 @@
 package org.sdase.commons.client.jersey;
 
 import com.fasterxml.jackson.databind.DeserializationFeature;
-import io.dropwizard.Application;
-import io.dropwizard.Configuration;
-import io.dropwizard.setup.Bootstrap;
-import io.dropwizard.setup.Environment;
+import io.dropwizard.core.Application;
+import io.dropwizard.core.Configuration;
+import io.dropwizard.core.setup.Bootstrap;
+import io.dropwizard.core.setup.Environment;
+import jakarta.ws.rs.client.Client;
+import jakarta.ws.rs.core.MediaType;
```

Comment:
What about lines 12 + 13 now?

Reply:
It was some missed lines that I forgot to remove. But it will be fixed in the next PR

Reply:
Should we take over [this test](https://github.com/SDA-SE/sda-spring-boot-commons/blob/master/sda-commons-dependency-check/src/test/java/org/sdase/commons/spring/boot/dependency/check/AvoidJavaxTest.java) from Spring Boot Commons, when we are a few steps further?

Reply:
Good idea, I will add in the next PR that includes Jakarta